docs/home.md

```diff
@@ -109,6 +109,11 @@ layout: documentation
 
 <div class="row">
   <div class="col-xs-6 col-md-4">
+    <a href="https://www.packtpub.com/big-data-and-business-intelligence/mastering-mesos" class="thumbnail">
+      <img src="https://www.packtpub.com/sites/default/files/6249OS_5186%20Mastering%20Mesos.jpg" alt="Mastering Mesos by Dipa Dubhashi and Akhil Das">
+    </a>
+<div class="row">
```

Comment:
Also it looks like you don't want to open a new class=row div here. You're missing the title text as well. If you have a look at the other entries you can see the expected format.